Description: Ensure Debian systems use endian.h.
 As of 2008-10-27, it seems that __GLIBC__ is no longer defined.
 Assume all Debian systems have <endian.h>.
Author: Steve Robbins <smr@debian.org>
Forwarded: no

--- boost-1.34.1.orig/boost/detail/endian.hpp
+++ boost-1.34.1/boost/detail/endian.hpp
@@ -30,7 +30,7 @@
 // GNU libc offers the helpful header <endian.h> which defines
 // __BYTE_ORDER
 
-#if defined (__GLIBC__)
+#if 1
 # include <endian.h>
 # if (__BYTE_ORDER == __LITTLE_ENDIAN)
 #  define BOOST_LITTLE_ENDIAN
